src/pages/signin/SignInPageLayout/SignInPageLayoutNarrow.js

```diff
@@ -19,46 +20,66 @@ const propTypes = {
     ...withLocalizePropTypes,
 };
 
-const SignInPageLayoutNarrow = props => (
-    <ScrollView
-        keyboardShouldPersistTaps="handled"
-        showsVerticalScrollIndicator={false}
-        style={[
-            styles.flexGrow1,
-            styles.signInPageNarrowContentContainer,
-            styles.alignSelfCenter,
-        ]}
-        contentContainerStyle={styles.ph5}
-    >
-        <KeyboardAvoidingView behavior="position">
-            <View style={[styles.flexGrow1]}>
-                <View
-                    style={[
-                        styles.signInPageInnerNative,
-                        styles.flex1,
-                        styles.dFlex,
-                        styles.flexColumn,
-                        styles.mt40Percentage,
-                    ]}
-                >
-                    <View style={[styles.componentHeightLarge, styles.mb2]}>
-                        <ExpensifyCashLogo
-                            width={variables.componentSizeLarge}
-                            height={variables.componentSizeLarge}
-                        />
+class SignInPageLayoutNarrow extends React.Component {
+    constructor(props) {
+        super(props);
+        this.form = null;
+    }
+
+    componentDidMount() {
+        // These native props are needed for Password Managers like LastPass
+        if (this.form) {
+            setNativePropsWeb(this.form, 'method', 'post');
+            setNativePropsWeb(this.form, 'action', '/');
```

Comment:
We should encapsulate this logic in a new kind of view then pass these as props. The component can then handle setting these native props with this `componentDidMount()` logic instead of having to repeat it everywhere we want to do this.

Reply:
Also please add some comments about why `method` and `action` are required in this case